mam/v2/sponge/sponge.h

```diff
@@ -68,29 +69,29 @@ typedef struct sponge_s {
  *
  * @return the trits
  */
-trits_t sponge_outer_trits(sponge_t const *const sponge);
+trits_t sponge_outer_trits(mam_sponge_t const *const sponge);
 
 /**
  * Sponge state initialization
  *
  * @param sponge Sponge interface
  */
-void sponge_init(sponge_t *const sponge);
+void sponge_init(mam_sponge_t *const sponge);
 
 /**
  * Internal state transformation
  *
  * @param sponge Sponge interface
  */
-void sponge_transform(sponge_t *const sponge);
+void mam_sponge_transform(mam_sponge_t *const sponge);
```

Comment:
Why is only this function getting a prefix ?

Reply:
sure, will rename all in a following PR